NetKAN/xScience.netkan

```diff
@@ -1,27 +1,18 @@
 {
     "spec_version": 1,
     "identifier": "xScience",
-    "$kref": "#/ckan/spacedock/234",
+    "$kref": "#/ckan/github/Flupster/KSP-X-Science",
     "license": "CC-BY-NC-SA-4.0",
     "author": ["Z-Key Aerospace", "Brodrick"],
-    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is held on vehicles and kerbals.  No matter what game scene you are in, what you current vehicle is, [x] Science! always reports on all science held anywhere.",
-    "resources": {
+    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is $    "resources": {
```

Comment:
You deleted some text and a new line here by accident

Reply:
I'm stupid! fixed, thanks for noticing